frontend/src/hooks/useAxios.ts

```diff
@@ -75,7 +75,7 @@ export const useAxios = () => {
      */
     const axiosInstance = axios.create({
       baseURL: handleApiUrl(),
-      timeout: 5000,
+      timeout: 10000,
```

Comment:
just be cautious, on updating this value, although i'm ok with it, we should understand that there is a valid reason it is taking 10s.

Reply:
So this report can be 56 pages long.

on the backend, it takes about 60 MS, but on the CDOGS server, we have a wait of a little over 5 seconds to render all those pages in a pdf and send it back.

The excel render takes a little less time for the same amount of pages


I can demonstrate this if you want